mycity/platforms/amazon/models/en_US.json

```diff
@@ -1133,6 +1133,15 @@
                         }
                     ]
                 },
+                {
+                    "name": "FoodTruckIntent",
+                    "slots": [],
+                    "samples": [
+                        "food trucks",
+                        "foodtrucks",
+                        "Nearby foodtrucks"
+                    ]
+                },
```

Comment:
This causes errors when uploading to the Alexa developer console. I think this is in the wrong section and should  be in the `intents` array above.